<?xml version="1.0"?>
<launch>
	<arg name="local_ip" default="192.168.25.218" />
	<!-- 第1台无人机 -->
	<group ns="/sunray_swarm/rmtt_1">
		<node pkg="rmtt_driver" name="rmtt_driver" type="rmtt_node.py" output="screen">
			<param name="drone_sn" type="string" value="0TQZM3QCNT00ZV" />
			<param name="drone_ip" type="string" value="192.168.25.28" />
			<param name="drone_port" type="string" value="8889" />
			<param name="local_ip" type="string" value="$(arg local_ip)" />
			<param name="local_port" type="string" value="8890" />
			<param name="video_port" type="string" value="11111" />
			<param name="enable_camera" type="bool" value="true" />
		</node>
		<!-- 启动rmtt_control_node -->
		<node pkg="sunray_swarm" type="rmtt_control_node" name="rmtt_control_node_1" output="screen">
			<param name="agent_id" value="1" />
			<param name="agent_ip" value="192.168.25.28" />
			<rosparam command="load" file="$(find sunray_swarm)/config/rmtt_params.yaml" />
		</node>
	</group>

</launch>
